Network/dedicated-load-balancer-license-management.md

```diff
@@ -74,6 +74,52 @@ The Netscaler Load Balancer uses FlexLM licensing to license features on the pro
   
 7. Observe the lines in the file that start with the word **INCREMENT** in all capital letters. This line contains the license expiration date.
 
+### <a name="update-license">Updating Licenses</a>
+
+CLC will provide the license file(s) with corresponding LB management and RNAT IP addresses. You or your team should have received the login credentials for your dedicated load balancer during its creation, but let us know if you need assistance with this.
+
+There are two methods depending on if you are updating a lone LB or an HA pair. Steps for each situation are below.
+
+#### HA Pair
+
+Users may need to [configure Java](../general/how-to-configure-java-settings-to-access-web-user-interfaces.md) to ensure they can access the Web GUI.
```

Comment:
your pull request failed to pass checks.  'General' must be capitalized I believe to pass